inttest/customports/customports_test.go

```diff
@@ -135,4 +136,15 @@ func (ds *Suite) TestControllerJoinsWithCustomPort() {
 
 	ds.T().Log("waiting to get logs from pods")
 	ds.Require().NoError(common.WaitForPodLogs(kc, "kube-system"))
+
+	// https://github.com/k0sproject/k0s/issues/1202
+	ds.T().Log("test that kubeconfig includes externalAddress")
```

Comment:
You could also wrap the following lines into their own sub-test via `ds.T().Run("kubeconfigIncludesExternalAddress", ...)` instead of logging it.

Reply:
I guess we should do so with the other `Log` calls too in that case? for consistency

Reply:
I see what you mean now. Yeah, that makes sense.